setup.go

```diff
@@ -0,0 +1,68 @@
+package main
```

Comment:
I'm leery of adding yet another language to the toolchain, especially one that doesn't come in a default Linux installation.

I'd be happier if this script was in Bash (there is a decent way to do command line parsing, see http://wiki.bash-hackers.org/howto/getopts_tutorial), or Python if you need to.